Doc/whatsnew/3.8.rst

```diff
@@ -743,12 +799,19 @@ They also re-appear in the box for the next customized run.  One can also
 suppress the normal Shell main module restart.  (Contributed by Cheryl
 Sabella, Terry Jan Reedy, and others in :issue:`5680` and :issue:`37627`.)
 
-Add optional line numbers for IDLE editor windows. Windows
+Added optional line numbers for IDLE editor windows. Windows
```

Comment:
Is there a clear choice on using past ("added") or present ("add") tense in the What's New or does it vary based on the entry? I typically see the present tense used more commonly.

Reply:
"Added" sounds better to my ear.